test.rb

```diff
@@ -0,0 +1,14 @@
+def greater_than_5 arr
+  count = 0
+
+  arr.count do |e| 
+    e > 5
```

Comment:
increment count if this evaluates to true:

```
 if ( e > 5) 
     count++
end 
```

Reply:
fixed issue and done.